schemas/test_cases/v0/misc.yaml

```diff
@@ -132,7 +132,15 @@
     - http://determined.ai/schemas/expconf/v0/profiling.json
   case:
     enabled: false
-    begin_on_batch: null
+    begin_on_batch: 0
+    end_after_batch: null
+
+- name: profiling is valid when only enabled is set
+  matches:
+    - http://determined.ai/schemas/expconf/v0/profiling.json
+  case:
+    enabled: true
+    begin_on_batch: 0
     end_after_batch: null
```

Comment:
You should delete these lines to actually test "when only enabled is set"

Reply:
Fixed. 

IMO this is sort of an undocumented DSL and it's not super easy to figure out how to use it correctly. We should add documentation on the DSL syntax.